modules/postgresql/metadata.display.yaml

```diff
@@ -111,24 +116,29 @@ spec:
         deletion_protection:
           name: deletion_protection
           title: Deletion Protection
+          altDefaults:
+            - type: ALTERNATE_TYPE_DC
+              value: false
         deletion_protection_enabled:
           name: deletion_protection_enabled
           title: Deletion Protection Enabled
           altDefaults:
             - type: ALTERNATE_TYPE_DC
-              value: true
+              value: false
```

Comment:
The default for this is already false. IIRC from prev discussions we wanted this to be true in alternate defaults?

Reply:
Yes! But this is causing friction when ADC users are modifying the name of sql instance! There application deployment is failing. 
The current ask is to disable it for now and enable at later stage!

Yes, the default is already false! But keeping it here as we would simply make it true when we want it at later stage!